File: testjavabenchmark/presentation/gui/stumbleupon/LinkGUI.java
package presentation.gui.stumbleupon;

import domain.stumbleupon.Link;
//Functional requirement 3.2.2
//Test case 165: All classes in package presentation.gui.stumbleupon are not allowed to use in a not direct lower layer
//Result: TRUE
public class LinkGUI {
	public LinkGUI(){
		//FR5.2
		System.out.println(Link.linkUrl);
	}
}

File: testprojects/csharp/benchmark/presentation/gui/stumbleupon/LinkGUI.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using CSharpBenchmark.domain.stumbleupon;

namespace CSharpBenchmark.presentation.gui.stumbleupon
{
    //Functional requirement 3.2.2
    //Test case 165: All classes in package presentation.gui.stumbleupon are not allowed to use in a not direct lower layer
    //Result: TRUE
    public class LinkGUI
    {
        public LinkGUI()
        {
            //FR5.2
            Console.WriteLine(Link.linkUrl);
        }
    }
}